smartling-files-api/src/main/java/com/smartling/api/files/v2/pto/ImportTranslationsPTO.java

```diff
@@ -0,0 +1,43 @@
+package com.smartling.api.files.v2.pto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import org.jboss.resteasy.annotations.providers.multipart.PartFilename;
+import org.jboss.resteasy.annotations.providers.multipart.PartType;
+
+import javax.ws.rs.FormParam;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class ImportTranslationsPTO
+{
+    @FormParam("file")
+    @PartType("application/octet-stream")
+    @PartFilename("file")
+    @NonNull
+    private byte[] file;
```

Comment:
Is there a way to do this without loading the entire file into memory? A quick Google search doesn't make it look possible. :(

Reply:
Actually it can be an InputStream, I can change it while it isn't released yet

Reply:
I think it must be a stream